service-catalog/components/pulp-operator.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: service-catalog-pulp-operator
+  title: Pulp Operator
+  description: Pulp is a platform for managing repositories of software packages and making them available to a large number of consumers
+  annotations:
+    operate-first.cloud/logo-url: https://docs.pulpproject.org/pulpcore/en/pulpproject.org_images/pulp_logo_big.png
+    backstage.io/kubernetes-label-selector: "control-plane=controller-manager"
```

This would show all pods matching this label in the cluster. I think you want to limit this to the Pulp operator only. I think following selector should work:

```suggestion
    backstage.io/kubernetes-label-selector: "control-plane=controller-manager,app.kubernetes.io/name=pulp-operator,olm.owner.namespace=opf-pulp"
```